proposals/libocispec.md

```diff
@@ -0,0 +1,37 @@
+# OCI libocispec project proposal
+
+## Abstract
+Provide a C library for parsing and accessing OCI Runtime and Image specification JSON.
+
+## Proposal
+With repositories under the https://github.com/opencontainers organization:
+
+Rename (transfer) https://github.com/giuseppe/libocispec to https://github.com/opencontainers/libocispec.
+
+### Initial Maintainers
+Initial maintainers of the libocispec project:
+* Giuseppe Scrivano <gscrivan@redhat.com> (@giuseppe)
+* Qiang Huang <h.huangqiang@huawei.com> (@hqhq)
+* Daniel J Walsh <dwalsh@redhat.com> (@rhatdan)
+
+### Code of Conduct
+This project would incorporate (by reference) the OCI Code of Conduct.
+
+### Governance and Releases
+This project would incorporate the Governance and Releases processes from the OCI project template: https://github.com/opencontainers/project-template.
+
+### Project Communications
+This project would continue to use existing channels in use by the OCI developer community for communication including:
+* GitHub for issues and pull requests
+* The dev@opencontainers.org email list
+* The monthly OCI developer community conference call
+* The #OpenContainers IRC channel
+
+### Versioning / Roadmap
+It aims to be a C library to be statically linked in the projects that need it.
```

Comment:
Guess that answers my versioning question :-)

Reply:
I guess I have to update this part :-) now libocispec supports also to be a shared library